pkg/apis/core/validation/validation_test.go

```diff
@@ -9602,627 +9602,823 @@ func TestValidatePod(t *testing.T) {
 	longPodName := strings.Repeat("a", 200)
 	longVolName := strings.Repeat("b", 60)
 
-	successCases := map[string]core.Pod{
+	successCases := map[string]struct {
+		spec core.Pod
+		opt  PodValidationOptions
```

Comment:
it looks unused?
Was this a leftover from a previous approach?

Reply:
Exactly, fixed.